vividus/src/main/java/org/vividus/bdd/steps/FileSteps.java

```diff
@@ -60,4 +60,16 @@ public void saveResponseBodyToFile(String name, String content, Set<VariableScop
         FileUtils.writeStringToFile(temporaryFile, content, StandardCharsets.UTF_8);
         bddVariableContext.putVariable(scopes, pathVariable, temporaryFile.getAbsolutePath());
     }
+
+    /**
+     * Saves content to file with specified pathname
+     * @param pathname Fully qualified file name with parent folders and extension (e.g. temp/some_file.txt)
+     * @param fileContent Content to be saved to file
+     * @throws IOException If an I/O error has occurred
+     */
+    @When("I create a file with the pathname `$pathname` and the content `$fileContent`")
```

Comment:
why not to add docs? :)

Reply:
this is a technical change: no business logic changes, no location changes
it's not hard for me to create the doc, but I'd like to keep such technical commits clear